### DISCUSSION TITLE
requiring "use all system resources" permission on Windows store?

### DISCUSSION THREAD

**Discussion by fantasyz** [ransome1/sleek#379] [General]
Hi, 

![ApplicationFrameHost_5wNghGhctf](https://user-images.githubusercontent.com/3223242/171800046-4efa5e0b-d6c2-4078-83b6-e3f858042105.png)

It it possible to reduce the app to just use the default UWP app permission where user only give file access right when they use the app to open a file?  I don't know if its possible because this is not a native UWP project where the permission requirement may be required by the tool that convert this Electron app to UWP app. 

Anyway this app looks amazing. It is exactly what I like a todo app to be.
I really want to give it a try but this permission requirement "use all system resources" (together with your github id ransome1) is a little scary.

Thanks

**Comment by ransome1** [comment]
I'm actually not declaring any permissions, it seems to be the default set of capabilities Electron Builder is adding to the manifest. When I find some time I can look into it. 

If you already know how set capabilities in the Electron Builder process for the appx target, let me know.

EDIT: I didn't look too deep into this, but there are some indication, that the Electron framework needs the full set of permissions in order to work: https://github.com/electron-userland/electron-builder/issues/3306. Like you kind of assumed.
